### PR TITLE
chore: bump version

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

**Comment by Himself65** [comment]
DONT MERGE

**Comment by himself65** [comment]
BREAKING CHANGE: https://github.com/vitejs/vite/pull/10762

File: packages/store/package.json [hunk 1]
```diff
@@ -20,14 +20,14 @@
     "buffer": "^6.0.3",
     "flexsearch": "0.7.21",
```
Comment (himself65) [inline packages/store/package.json:21]: BREAKING CHANGE here. no time to upgrade

**Review by doodlewind — APPROVED** [review]
LGTM, thanks for upgrading the toolchain!

**Pull request merged**